### DISCUSSION TITLE
commit problem

### DISCUSSION THREAD

**Discussion by ppssr143** [community/community#22348] [New to GitHub]
<p>i need a small problem</p>
<p>i have create files and add some lines in to that files i will push the code and in to the reposotrie and added some lines in to the files and push the code in to the reposotries now my proble with out clone the code in to the reposotries is there any possibility to the revert back in to the previous commits</p>
<p>how can i overcome to this proble</p>

**Comment by i-marsh** [comment, marked as answer]
<p>Take a look at git revert and git reset - there are a number of different approaches to returning to a previous state / commit</p>
<p> - <a href="https://git-scm.com/docs/git-revert.html" rel="nofollow noopener">Git Revert Documentation</a></p>
<p> - <a href="https://git-scm.com/docs/git-reset.html" rel="nofollow noopener">Git Reset Documentation</a> </p>
